js/index.js

```diff
@@ -20,7 +20,7 @@ INSTRUCTIONS:
     project, when tests are failing, 
     you should get helpful errors 
     along the way!
-    ************/
```

Comment:
I suggest you add a professional read me file, use the template from [here](https://github.com/microverseinc/readme-template/blob/master/README.md)
Remember that a ReadMe file should contain important information about your project like (description about it, how to set up the project, authors and aditional indications if needed).